src/main/java/separator/Separator.java

```diff
@@ -0,0 +1,67 @@
+package separator;
+
+import java.util.ArrayList;
+
+public class Separator {
+    public static ArrayList<Character> separators = new ArrayList<>();
+
+    public Separator() {
+        separators.add(',');
+        separators.add(':');
+    }
+
+    public void saveSeparator(String input) {
+        if (!isvalidatedCustomSeparatorForParsing(input)) {
+            return;
+        }
+
+        Character customSeparator = parseCustomSeparator(input);
+        addNewSeparator(customSeparator);
+    }
+
+    public void addNewSeparator(Character separator) {
+        if (separator == null) {
+            return;
+        }
+        separators.add(separator);
+    }
+
+    public Character parseCustomSeparator(String string) {
+        Character customSeparator = null;
+
+        if (string.substring(0, 4).equals("//\\n")) {
+            return customSeparator;
+        }
+
+        customSeparator = string.charAt(2);
+
+        return customSeparator;
+    }
+
+    public boolean isvalidatedCustomSeparatorForParsing(String string) {
```

Comment:
```suggestion
    public boolean isValidateCustomSeparatorForParsing(String string) {
```
이렇게 쓰는게 변수명에 조금 더 적합해보이는데 어떻게 생각하세요? 변수명에는 현재형으로 쓰는거라고 얼핏 봤던 것 같아요..!

Reply:
흑 죄송합니다..ㅠ 제가 영어가 많이 약해서 `be동사`+`동사ed` 이렇게 넣는게 맞다 생각했습니다..!